kotlin/change-viewpoint/README.md

```diff
@@ -0,0 +1,44 @@
+# Change viewpoint
+
+Set the map view to a new viewpoint.
+
+![Image of change viewpoint](change-viewpoint.png)
+
+## Use case
+
+Programmatically navigate to a specified location in the map or scene. Use this to focus on a particular point or area of interest.
+
+## How to use the sample
+
+The map view has several methods for setting its current viewpoint. Select a viewpoint from the UI to see the viewpoint changed using that method.
```

Comment:
Of course this will change if the UI changes, but it may be better worded something like "Select a viewpoint option to see the map view move to that location" or similar.